modules/rosa-hcp-sts-accessing-a-break-glass-cred-cli.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-ext-auth.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-accessing-a-break-glass-cred-cli_{context}"]
+= Accessing a {hcp-title} cluster by using a break glass credential
+
+Use the new `kubeconfig` from the break glass credential to gain temporary admin access to a {hcp-title} cluster.
+
+.Prerequisites
+
+* You have access to a {hcp-title} cluster with external authentication enabled. For more information, see _Creating {hcp-title} clusters with external authentication_.
+* You have installed the `oc` CLI.
+
+.Procedure
+
+. Optional: Create a `kubeconfigs` directory:
```

Comment:
This is not an optional command, if the customer doesn't have created kubeconfig in their home directory, then they will get this error when running 3rd step of this procedure
`$ rosa describe break-glass-credential 2b0lpih06ndmgc252jnjhj8j71dnc5uu -c aaraj-hcp --kubeconfig > $KUBECONFIG
bash: /home/aaraj/kubeconfigs/break-glass-aaraj-hcp.kubeconfig: No such file or directory
`